### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by chevtche**

### CODE REVIEW

**Comment by eile** [comment]
+1

**Comment by eile** [comment]
+1

File: tests/CMakeLists.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Copyright (c) HBP 2015 Daniel.Nachbaur@epfl.ch
 # Change this number when adding tests to force a CMake run: 0
 
-set(TEST_LIBRARIES ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY} zerobuf)
+set(TEST_LIBRARIES ${Boost_UNIT_TEST_FRAMEWORK_LIBRARY} ZeroBuf)
```
Comment (tribal-tec) [inline tests/CMakeLists.txt]: Why? It's common_library(zerobuf)

File: zerobuf/CMakeLists.txt [hunk 2]
```diff
@@ -30,10 +28,12 @@ set_source_files_properties(jsoncpp/jsoncpp.cpp PROPERTIES COMPILE_FLAGS
   -Wno-shadow)
 
 set(ZEROBUF_LINK_LIBRARIES ${SERVUS_LIBRARIES})
-set(PROJECT_INCLUDE_NAME zerobuf)
+set(ZEROBUF_INCLUDE_NAME zerobuf)
 set(ZEROBUF_OMIT_LIBRARY_HEADER ON)
-common_library(zerobuf)
+common_library(ZeroBuf)
```
Comment (tribal-tec) [inline zerobuf/CMakeLists.txt]: Ah. Ok then.

**Comment by chevtche** [comment]
Can this be merged ?

**Pull request merged**